app/services/account_reset/delete_account.rb

```diff
@@ -66,13 +66,19 @@ def notify_user_via_email_of_deletion
     end
     # rubocop:enable IdentityIdp/MailLaterLinter
 
+    def profile_components
+      user.active_profile.proofing_components if user.active_profile?
+    end
+
     def extra_analytics_attributes
       {
         user_id: user.uuid,
         email: user.email_addresses.take&.email,
         account_age_in_days: account_age,
         account_confirmed_at: user.confirmed_at,
         mfa_method_counts: mfa_method_counts,
+        proofing_components: profile_components,
```

Comment:
I hate to switch it up at this point, but I'm thinking it may be better off to just use (or at least start with using) the `Profile#idv_level` value ([mentioned in Slack](https://gsa-tts.slack.com/archives/C03DCUECFH7/p1726596330189789?thread_ts=1726594700.506499&cid=C03DCUECFH7)). Looking at some real-world logging, proofing components alone can't tell us if the user was proofed with biometric. Conversely, `idv_level` can't tell us if they proofed with GPO, but I think it's probably more pertinent to the event that we know "legacy" vs. with biometric vs in-person.

Plus, strings are a bit easier to work with than a hash 😅 

```suggestion
        profile_idv_level: user.active_profile&.idv_level,
```

Reply:
That worked great. Thanks also to @matthinz for the suggestion.